examples/alice_dio/lib/main.dart

```diff
@@ -19,11 +20,8 @@ class MyApp extends StatefulWidget {
 class _MyAppState extends State<MyApp> {
   late final AliceDioAdapter _aliceDioAdapter = AliceDioAdapter();
 
-  late final Alice _alice = Alice(
-    showNotification: true,
-    showInspectorOnShake: true,
-    maxCallsCount: 1000,
-  )..addAdapter(_aliceDioAdapter);
+  late final Alice _alice = Alice(configuration: AliceConfiguration())
```

Comment:
Since this is an example, declare the configuration in a separate variable, just so that it's clearer for a new user.

Reply:
👍